nucypher/blockchain/eth/interfaces.py

```diff
@@ -232,6 +232,10 @@ def __init__(self,
         self.client = NO_BLOCKCHAIN_CONNECTION
         self.transacting_power = READ_ONLY_INTERFACE
         self.is_light = light
+
+        # FIXME: Not ready to give users total flexibility. Let's stick for the moment to known values. See #2447
+        if gas_strategy not in ('slow', 'medium', 'fast', 'free', None):
```

Comment:
Can we use key values from `WEB3_GAS_STRATEGIES.keys()` to prevent us from having to hardcode `'slow', 'medium', 'fast'`?